net/modemmanager/Makefile

```diff
@@ -0,0 +1,135 @@
+PKG_HASH=fd0f39996025dac96995daea8a58ec7dd571582f7563a8ae0d5f65b571b76ee2
+
+PKG_MAINTAINER:=Nicholas Smith <nicholas.smith@telcoantennas.com.au>
+PKG_LICENSE:=GPL-2.0-or-later
+PKG_LICENSE_FILES:=COPYING COPYING.LIB
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/ModemManager-$(PKG_VERSION)
+HOST_BUILD_DEPENDS:=libglib2.0-dev/host libtool/host gtk-doc-tools/host libgudev-1.0-dev/host
+BUILD_PARALLEL:=1
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/nls.mk
+
+define Package/modemmanager
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=Control utility for any kind of mobile broadband modem
+  URL:=https://www.freedesktop.org/wiki/Software/ModemManager
+  DEPENDS:= \
+	$(INTL_DEPENDS) \
+	+glib2 \
+	+libqmi \
+	+libmbim \
+	+dbus \
+	+usb-modeswitch \
+	+ppp \
+	+kmod-usb-net \
+	+kmod-usb-net-cdc-ncm \
+	+kmod-usb-net-huawei-cdc-ncm \
+	+kmod-usb-net-cdc-mbim \
+	+kmod-usb-net-qmi-wwan \
+	+kmod-usb-net-hso \
+	+kmod-usb-net-rndis \
+	+kmod-usb-net-sierrawireless \
+	+kmod-usb-acm \
+	+kmod-usb-wdm \
+	+kmod-usb-serial \
+	+kmod-usb-serial-option \
+	+kmod-usb-serial-qualcomm \
+	+kmod-usb-serial-sierrawireless \
+	+kmod-usb-serial-option \
+	+kmod-usb-serial-wwan
+endef
+
+define Package/modemmanager/description
+  ModemManager is a DBus-activated service which allows controlling mobile
+  broadband modems.
+endef
+
+CONFIGURE_ARGS += \
+	--with-qmi \
+	--with-mbim \
+	--without-polkit \
+	--without-udev \
+	--without-suspend-resume \
+	--with-systemdsystemunitdir=no \
+
+define Build/Prepare
+	$(call Build/Prepare/Default)
+	( cd "$(PKG_BUILD_DIR)"; \
+		echo -e "all:\ninstall:" >po/Makefile.in.in; \
+	)
+endef
+
+define Build/InstallDev
+	$(INSTALL_DIR) $(1)/usr/include/ModemManager
+	$(CP) $(PKG_INSTALL_DIR)/usr/include/ModemManager/*.h $(1)/usr/include/ModemManager
+	$(INSTALL_DIR) $(1)/usr/include/libmm-glib
+	$(CP) $(PKG_INSTALL_DIR)/usr/include/libmm-glib/*.h $(1)/usr/include/libmm-glib
+	$(INSTALL_DIR) $(1)/usr/lib
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libmm-glib.so* $(1)/usr/lib
+	$(INSTALL_DIR) $(1)/usr/lib/pkgconfig
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/pkgconfig/ModemManager.pc $(1)/usr/lib/pkgconfig
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/pkgconfig/mm-glib.pc $(1)/usr/lib/pkgconfig
+endef
+
+define Package/modemmanager/install
+	$(INSTALL_DIR) $(1)/lib/udev/rules.d
+	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/lib/udev/rules.d/*.rules $(1)/lib/udev/rules.d
+
+	$(INSTALL_DIR) $(1)/usr/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/ModemManager $(1)/usr/sbin
+
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/mmcli $(1)/usr/bin
+
+	$(INSTALL_DIR) $(1)/usr/lib
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libmm-glib.so* $(1)/usr/lib
+
+	$(INSTALL_DIR) $(1)/usr/lib/ModemManager
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/ModemManager/libmm-plugin-*.so* $(1)/usr/lib/ModemManager
+
+	$(INSTALL_DIR) $(1)/etc/dbus-1/system.d
+	$(CP) $(PKG_INSTALL_DIR)/etc/dbus-1/system.d/org.freedesktop.ModemManager1.conf $(1)/etc/dbus-1/system.d
```

Comment:
Hm... why not install this config to /usr/share/dbus-1/system.d ? I thought that /etc hierarchy is more for admin purpose and /usr/share hierarchy is for operating system installed stuff. I think the later would fit our shipped file... However, this is a not a strong opinion I think many packages are still packaged this way...

Reply:
BTW: why not $(INSTALL_DATA) here?